provider/v1alpha1/service.proto

```diff
@@ -59,6 +61,23 @@ message MountRequest {
 message MountResponse {
     repeated ObjectVersion object_version = 1;
     Error error = 2;
+    // files contains the entire mount volume filesystem.
+    repeated File files = 3;
+}
+
+// File holds secret file contents and location in the mount path to write the
+// file.
+message File {
```

Comment:
Could we also add a comment with the limitations on the size of the payload?

Reply:
I'm assuming bigger size than default supported would mean we would need to use grpc stream? I would open to keeping the default with the limitation documented. If there are users with additional size, then we can add an option to configure. WDYT?

Reply:
I updated the `repeated File files = 3` comment but we can just add a flag to control the [`MaxCallRecvMsgSize`](https://pkg.go.dev/google.golang.org/grpc#WithMaxMsgSize) option when creating the plugin grpc client. This would require config/deploy from the user but would not require switching to grpc streaming. I believe this was what @tomhjp preferred in the design doc